packages/components/buttons/secondary-button/src/secondary-button.tsx

```diff
@@ -55,6 +59,10 @@ export type TSecondaryButtonProps<
   onClick?: (
     event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
   ) => void;
+  /**
+   * Indicates the size of the button.
+   */
+  size?: 'small' | 'medium' | 'big';
```

Comment:
Since we are already deprecating the `small` value int he `PrimaryButton` I think we could just not use it in this component.

Reply:
That makes more sense, thanks! [93b4501](https://github.com/commercetools/ui-kit/pull/2443/commits/93b45011dc06a1efcb078fa61823968cb7182eb7).